web/components/Canvas/MainMenu/MainMenu.module.scss

```diff
@@ -1,28 +1,27 @@
 .MainMenu {
   display: flex;
   justify-content: center;
-  align-items: center;
-  height: 100%;
+  margin-top: 50%;
+  min-width: 700px;
+  min-height: 500px;
 
-  .buttons {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
+  .lineBreak {

```

Comment:
unused class

Reply:
Good catch, I forgot I removed the line break. It's now updated.